packages/protocol/contracts/governance/Election.sol

```diff
@@ -467,6 +469,76 @@ contract Election is IElection, Ownable, ReentrancyGuard, Initializable, UsingRe
     return votes.total.eligible.contains(group);
   }
 
+  /**
+   * @notice Returns the amount of rewards that voters for `group` are due at the end of an epoch.
+   * @param group The group to calculate epoch rewards for.
+   * @param totalEpochRewards The total amount of rewards going to all voters.
+   * @return The amount of rewards that voters for `group` are due at the end of an epoch.
+   * @dev Eligible groups that have received their maximum number of votes cannot receive more.
+   */
+  function getGroupEpochRewards(
+    address group,
+    uint256 totalEpochRewards
+  )
+    external
+    view
+    returns (uint256)
+  {
+    bool meetsBalanceRequirements = (
+      getLockedGold().getAccountTotalLockedGold(group) >=
+      getValidators().getAccountBalanceRequirement(group)
```

Comment:
this means that if somehow parameters changed, and now the group does not meet balance requirements; his rewards are going to be divided on the other groups.

I think that's ok, but maybe deserves a comment

Reply:
Ok, so, the block rewards would be less then. For the blockchain. 

Does that needs to be considered on our monetary policy? I mean, probable there's an assumption of constant block rewards, but this creates a condition where that's not true. It can be less than that. 

Again, not saying this is a problem, just asking if this is something that needs to be considered